packages/css/src/transformCss.ts

```diff
@@ -111,10 +126,10 @@ class Stylesheet {
     this.conditionalRulesets = [new ConditionalRuleset()];
     this.fontFaceRules = [];
     this.keyframesRules = [];
-    this.localClassNameRegex =
-      localClassNames.length > 0
-        ? RegExp(`(${localClassNames.map(escapeStringRegexp).join('|')})`, 'g')
-        : null;
+    this.localClassNamesMap = new Map(
```

Comment:
If the key and value are the same, and the classname is just being used for a lookup, could this not be a `Set` instead?

Reply:
Answer from Matt: `Map` is faster in this case